kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -290,7 +292,10 @@
         } else if (option === Options.REGISTER) {
           this.facilityForRegister = facility;
         } else if (option === Options.MANAGESYNC) {
-          this.$router.push({ path: '/manage' });
+          this.$router.push({
+            name: SyncPageNames.MANAGE_SYNC_SCHEDULE,
+            params: { facility_id: facility.id },
```

Comment:
I think this should be `facilityId` and not `facility_id` as device plugin uses the former

Reply:
@akolson see my note above, as we still have code that is referencing the route parameters within both plugins, we might be better off just using `facility_id` across both plugins so that it works, even if it's not the cleanest separation of concerns.